network/kademlia_load_balancer.go

```diff
@@ -0,0 +1,244 @@
+// Copyright 2019 The Swarm Authors
```

Comment:
Moving to a different package will be hard, as I am using these tests functions in network pacakge:

- testKadPeerAddr
- newTestKademlia

Reply:
I know, good design is not easy, and this is a sign that there is a problem with modularity. I do not insist, but I want to point to the red flag.

Reply:
The thing is that the testing of kademlia is hard, maybe we should move those functions to a network/kademliatest so it can be used from elsewhere. And also, it seems that kademlia should be in its own package.

Reply:
I agree completely. This could be a good improvement in the future.